features/support/paths.rb

```diff
@@ -575,6 +587,9 @@ def path_to(page_name, *args)
     when "finance settings"
       admin_finance_settings_path
 
+    when "the finance settings"
+      admin_finance_settings_path
```

Comment:
I don't think we need this twice 🤔 It is the same as the previous one. So maybe we should remove the previous one and rename them all to "the finance settings"

Reply:
that will break other tests so I'm gonna leave it.

Reply:
well we should make the 'the' optional

Reply:
@thomasmaas It shouldn't break anything, it should be a very straightforward change 🤔 Do you want me to do it quickly and check it doesn't break anything (through Jenkins/CircleCI) ?